apps/web/src/lib/reference/asset-reference-config.ts

```diff
@@ -0,0 +1,22 @@
+export const ASSET_REFERENCE_TYPE_INFO = {
+    asset_compliance_type: {
+        type: 'asset_compliance_type',
+        key: 'compliance_type',
+        name: 'Compliance Type',
+    },
+    asset_account: {
+        type: 'asset_account',
+        key: 'account',
+        name: 'AWS Account ID (Asset)',
+    },
+    asset_compliance_number: {
+        type: 'asset_compliance_number',
+        key: 'compliance_number',
+        name: 'Compliance Number',
+    },
+    asset_service: {
+        type: 'asset_service',
+        key: 'service',
+        name: 'Service',
+    },
```

Comment:
question: Oh, I thought we have only two asset variables. Are they added?

Reply:
These are not included in the **Asset managed variable**, but they are part of the **Group by (Asset)** list.
![스크린샷 2023-05-25 오후 1 43 29](https://github.com/cloudforet-io/console/assets/18563857/9c4e49ad-7c1a-4f44-a7f2-2c55ef13e4c1)